src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicWithHorizon.hpp

```diff
@@ -0,0 +1,459 @@
+// Distributed under the MIT License.
```

Comment:
I didn't diff it, but it looks like this file is the same as what EvolveGeneralizedHarmonic.hpp used to be (i.e., it includes all the stuff deleted from that file here). Is that correct?

If so, is there a reason to just have two different hpp files for GH evolutions (one with, one without, a horizon finder) like this, instead of using the `EvolveGeneralizedHarmonicBase.hpp` etc. refactoring in the branch you made for my BBH runs? 

https://github.com/moxcodes/spectre/tree/cce_gh_exe_and_bjorhus_bc/src/Evolution/Executables/GeneralizedHarmonic

I'm just wondering why not implement this kind of refactoring, and then add the "no horizon" case as one more example case, to minimize the duplicated code. That would also make life easier for my bbh branches, by reducing the magnitude of the changes from develop and the likelihood of conflicts

Reply:
That's a reasonable point -- the only reason I didn't perform the factoring now was that I wasn't yet certain whether there's consensus on how similar metavariables like this should be factored.
I'll figure out if anyone has strong feelings against, and if not, I'll do the factoring with inheritance.